lib/rake/test.rake

```diff
@@ -120,12 +120,10 @@ namespace :test do
     Dir.chdir(dashboard_dir) do
       ChatClient.wrap('dashboard ruby unit tests') do
         ENV['DISABLE_SPRING'] = '1'
-        ENV['UNIT_TEST'] = '1'
```

Comment:
safe to remove because we already set `ENV['UNIT_TEST']` in `dashboard/test/test_helper.rb`, before the rails environment is loaded. This config/codepath is already vetted because dashboard unit tests are run in CI without explicitly setting `ENV['UNIT_TEST']` from within the rake task: https://github.com/code-dot-org/code-dot-org/blob/1582c1dc51c5ccff33c285f688a09bf3ebdd7bb6/lib/rake/test.rake#L280-L282